packages/pigeon/unit_tests/ios_unit_tests/ios/Runner/messages.h

```diff
@@ -0,0 +1,37 @@
+// Autogenerated from pigeon, do not edit directly.
+#import <Foundation/Foundation.h>
```

Comment:
Do you want to check these files in rather than re-generating them each time? This increases the likelihood of someone running the tests against stale outputs

Reply:
done